index.html

```diff
@@ -1,139 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link  rel="stylesheet" href="styles/style.css">
-    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;600;700;800&display=swap" rel="font">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;600;700;800&display=swap"
+      rel="font"
+    />
     <title>Drivers Club</title>
+  </head>
+  <body>
+    <section class="landing-section">
+      <div class="landing-section-header">
+        <span>Driver's club.</span>
+        <div class="box-horizontal"></div>
+        <a href="#about">About us</a>
+      </div>
 
-</head>
-<body>
- <!--Landing section-->
-    <div class="landing-section">
-        <div class="container-1-pc">
-            <div id="landing-heading"><h1>Driver's club.</h1></div>
-            <div class="box-horizontal"></div>
-            <div id="about-us"><h1><a href="#about">About us</a></h1></div>
-        </div>
+      <div class="landing-section-middle">
+        <div class="box-vertical"></div>
+        <div class="box-vertical"></div>
+      </div>
 
-        <div class="box-container">
-            <div class="box-vertical"></div>
-            <div class="box-vertical"></div>
-        </div>
+      <div class="landing-section-footer">
+        <a href="#carss">Cars</a>
+        <div class="box-horizontal"></div>
+        <a href="#contact">Sign up</a>
+      </div>
+    </section>
 
-        <div class="container-2-pc">
-            <div id="cars"><h1><a href="#carss">Cars</a></h1></div>
-            <div class="box-horizontal" id="box-horizontal-2"></div>
-            <div id="sign-up"><h1><a href="#contact">Sign up</a></h1></div>
-        </div>
-        <!--Mobile-->
-        <div class="mobile">
-            <div class="mobile-heading">
-                <h1 id="mobile">Driver’s Club.</h1>
-            </div>
-            <div class="mobile-about">
-                <h1>About us</h1>
-            </div>
-            <div class="mobile-cars">
-                <h1>Cars</h1>
-            </div>
-            <div class="mobile-sign-up">
-                <h1>Sign up</h1>
-            </div>
-        </div>
```

Comment:
Insanity.

Reply:
I will try to avoid this in the future, thank you for the feedback.